### PR TITLE
Material icon medium dartpad demo

### PR DESCRIPTION
**Pull request by JoseAlba**
#### Don't forget:
- [x] Are you sure you mean this repo? This repository is a reference to the [main Flutter repository](https://github.com/flutter/flutter). Code-level issues should be created there.
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [x] Sign the CLA bot. You can do this once the pull request is opened.

Be sure to read the [Flutter guide on contributing](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md).

### Related issues
 - Type `#` and the issue number or `#` and select the from the inline drop down.

### CODE REVIEW

File: medium/material_icons/dartpad_metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: Material Icons Demonstration
+mode: flutter
+files:
+  - name: main.dart
```
Comment (rami-a) [inline medium/material_icons/dartpad_metadata.yaml:4]: Add new line

**Review by johnsonmh — APPROVED** [review]

File: medium/material_icons/main.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      title: 'Material Icon Demo',
+      home: MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Row(
```
Comment (rami-a) [inline medium/material_icons/main.dart:18]: You might want to put it in a scaffold so that it has a white background

**Review by rami-a — APPROVED** [review]

File: medium/material_icons/dartpad_metadata.yaml (hunk 1 above)
Comment (pennzht) [inline medium/material_icons/dartpad_metadata.yaml:4]: Maybe add a newline? Git shows "no newline at end of file"

**Review by pennzht — APPROVED** [review]

File: medium/material_icons/dartpad_metadata.yaml (hunk 1 above)
Comment (JoseAlba) [reply medium/material_icons/dartpad_metadata.yaml:4]: Done

Comment (JoseAlba) [reply medium/material_icons/dartpad_metadata.yaml:4]: Done

File: medium/material_icons/main.dart (hunk 2 above)
Comment (JoseAlba) [reply medium/material_icons/main.dart:18]: Done

**Pull request merged**